cfme/common/vm.py

```diff
@@ -297,10 +297,15 @@ def find_quadicon(
         # this is causing some issues in 5.5.0.9, commenting out for a bit
         # paginator.results_per_page(1000)
         if use_search:
-            if not search.has_quick_search_box():
-                sel.force_navigate('vm_templates_provider_branch',
-                                   context={'provider_name': self.provider.name})
-            search.normal_search(self.name)
+            try:
+                if not search.has_quick_search_box() and self.provider.instances_page_name:
+                    # We don't use provider-specific page (vm_templates_provider_branch) here
+                    # as those don't list archived/orphaned VMs
+                    sel.force_navigate(self.provider.instances_page_name)
```

Comment:
Just thinking - there is a ``do_not_navigate`` - should we take that into account here too?